Add setting doc for auto disk provisioning and ssl parameter (#84)
harvester/docs: docs/settings/settings.md

```
ens3
```

## `auto-disk-provision-paths`

This setting allows Harvester to automatically add disks that match the given glob pattern as VM storages.
It's possible to provide multiple patterns by separating them with comma.

!!! warning
    - This setting is applied to **every Node** in the cluster.
    - All the data in these storage devices **will be destroyed**. Use at your own risk.

Default: none

#### Example

> Add disks matches glob pattern `/dev/sd*` or `/dev/hd*`:

Reviewer: ```suggestion
The following example, will add disks matching the glob pattern `/dev/sd*` or `/dev/hd*`:
```

```
/dev/sd*,/dev/hd*
```

## `ssl-parameters`

This setting allows you to change the enabled SSL/TLS protocols and ciphers of Harvester GUI and API.

The following options and values can be set:

- `protocols`: Enabled protocols. See NGINX Ingress Controller's configs [`ssl-protocols`](https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/configmap/#ssl-protocols) for supported input.

- `ciphers`: Enabled ciphers. See NGINX Ingress Controller's configs [`ssl-ciphers`](https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/configmap/#ssl-ciphers) for supported input.

> If no value is provided, default protocols are set to `TLSv1.2` only and ciphers are
> `ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305`.

Reviewer: ```suggestion
If no value is provided, `protocols` is set to `TLSv1.2` only and the `ciphers` list is
`ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305`.
```

Reviewer: for the wording `cyphers list`, I checked the NGINX page referenced

Default: none

#### Example

> Set SSL/TLS enabled protocols to `TLSv1.2` and `TLSv1.3` and enabled ciphers to
> `ECDHE-ECDSA-AES128-GCM-SHA256` and `ECDHE-ECDSA-CHACHA20-POLY1305`.

Reviewer: ```suggestion
The following example sets the enabled SSL/TLS protocols to `TLSv1.2` and `TLSv1.3` and the ciphers list to
`ECDHE-ECDSA-AES128-GCM-SHA256` and `ECDHE-ECDSA-CHACHA20-POLY1305`.
```

```
ens3
```

## `auto-disk-provision-paths`

> This setting allows Harvester to automatically add disks that match the given glob pattern as VM storages.

Reviewer 2: \s\storages\storage

> It's possible to provide multiple patterns by separating them with comma.

Reviewer 2: \s\comma\a comma

!!! warning
    - This setting is applied to **every Node** in the cluster.
    - All the data in these storage devices **will be destroyed**. Use at your own risk.

Default: none

#### Example

> The following example, will add disks matching the glob pattern `/dev/sd*` or `/dev/hd*`:

Reviewer 2: remove the `,` after example
